index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Luis Sanchez</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html"><h1 class="name">Luis Sanchez</h1></a>
+    <nav class="navigator">
+      <img src="images/menu.png" alt="menu button">
+
+    </nav>
+  
+  </header>
+  <section class="headline">
+    <h1 class="introduction">Hey There.
+      I'm Luis
+    </h1>
+    <h1 class="supporting-text">I'm a software developer</h1>
```

Comment:
- [ ] [OPTIONAL] It is considered a best practice to have a single `h1` element per page. I highly recommend you replace the second `h1` element with an appropriate `hx` tag. (i.e. `h2`, `h3`,...)
Kindly check [this article](https://www.reliablesoft.net/h1-tag/) to get an insight into how to utilize `h1` element effectively. 🤝️